localtoast_sql/localtoast_sql.go

```diff
@@ -60,26 +69,62 @@ func (a *localScanAPIProvider) FilePermissions(ctx context.Context, filePath str
 	return localfilereader.FilePermissions(ctx, a.fullPath(filePath))
 }
 
+func (a *localScanAPIProvider) SupportedDatabase() (ipb.SQLCheck_SQLDatabase, error) {
```

Comment:
Can we support multiple DB types in the same session? AFAICS if this function was something like
`func IsDatabaseSupported(ipb.SQLcheck_SQLDatabase) bool`
Then we could return true for both Cassandra and MySQL if they're both specified in the CLI
And then we can accept checks both of type SQLCheck_DB_MYSQL and SQLCheck_DB_CASSANDRA in the same run.

Reply:
I guess the code is a bit simpler this way and we don't have use cases where we want to use both databases at the same time, so we can leave it as is.